gulp/tasks/watch.js

```diff
@@ -4,8 +4,12 @@
 const gulp = require( 'gulp' );
 const config = require( '../config' );
 
-gulp.task( 'watch', [ 'browsersync' ], function() {
-  gulp.watch( config.scripts.src, [ 'scripts' ] );
-  gulp.watch( config.styles.cwd + '/**/*.less', [ 'styles' ] );
-  gulp.watch( config.images.src, [ 'images' ] );
-} );
+gulp.task( 'watch',
+  gulp.series(
```

Comment:
Why make this series, here? Doesn't seem like it was before.

Reply:
I think it was? Before it defined that the `browsersync` task should run before the anonymous function when running `gulp watch`, and that's how it's set up with series.